src/description/eth_call.md

```diff
@@ -0,0 +1,11 @@
+* **MUST NOT** affect on-chain state 
+
+* **MUST** allow a `from` parameter that does not exist on-chain and if `from` is not defined it **MUST** be interpreted as `0x0000000000000000000000000000000000000000`
```

Comment:
What happens if the from account is not an EOA?

Reply:
Playing around with different from addresses I found that as long as it a 20 byte hex string the behavior is the same